packages/contact-center/cc-components/src/components/StationLogin/station-login.tsx

```diff
@@ -78,12 +87,21 @@ const StationLoginComponent: React.FunctionComponent<StationLoginComponentProps>
 
   const updateTeam = useCallback(
     (value: string) => {
-      setTeamsValue(value);
       setTeam(value);
     },
     [setTeam]
   );
 
+  const updateDialNumberLabel = (selectedOption: string): void => {
+    if (selectedOption === 'AGENT_DN') {
+      setDialNumberLabel('Dial number');
+      setDialNumberPlaceholder('Dial Number');
```

Comment:
From the Figma, this should be **Dial number**

```suggestion
      setDialNumberPlaceholder('Dial number');
```

Also do we need two label constants for this? The text seems to be same for both. Can we move to single variable?